routes/books.js

```diff
@@ -0,0 +1,81 @@
+var express = require('express'),
+    router = express.Router();
+var fs = require("fs");
+var bookSchema = require('../models/books');
```

Comment:
Couple of things here
1. Use `const/let` instead of `var` (Same applies for all the other files in this pull request)
2. Try avoiding declaring multiple variables on the same line as that is not recommended as a good practice